plugin/DataView/DataDetailView.php

```diff
@@ -117,19 +116,6 @@ public function __construct()
 		$this->_pDataViewSimilarEstates = new DataViewSimilarEstates();
 	}
 
-
-	/**
-	 *
-	 */
-
-	public function __wakeup()
```

Comment:
I think this should be pulled back in since `DataViewSimilarEstates` is needed for the migration from database version 16 to 17.